src/HalfIntegers.jl

```diff
@@ -23,6 +23,7 @@ export
     # Functions
     half,
     ishalfinteger,
+    ishalfoddinteger,
```

Comment:
Should I rename this function to `ishalfodd`, like `isinteger` and `isodd`?

Reply:
Thank you for the review! I named the function referring to the word "half-odd-integers" in the Wikipedia article.
But I now think consistency with `Base.odd` and `Base.isinteger` has priority, so I will update the name. (I think `ishalfodd` is still clear enough.)

Reply:
I don’t understand this. To me, `ishalfodd` seems consistent with `Base.isodd` and `ishalfoddinteger` seems consistent with `Base.isinteger`. Why do you think that `ishalfoddinteger` is more consistent with both of them?

Reply:
Sorry for confusing. I was just trying to clarify the pros and cons of these function names. The reason of `ishalfoddinteger` was just borrowing from the wikipedia article. I now changed the name to `ishalfodd` which is consistent with Base.

Reply:
https://github.com/sostock/HalfIntegers.jl/pull/59#discussion_r1415705222 was written before the commit b0bc2e5096609c3030331f171184b7713f3547f7.